code/modules/antagonists/wizard/equipment/spellbook_entries/offensive.dm

```diff
@@ -85,6 +85,19 @@
 	category = "Offensive"
 	no_coexistance_typecache = list(/datum/action/cooldown/spell/lichdom)
 
+/datum/spellbook_entry/sanguine_strike
+	name = "Exsanguinating Strike"
+	desc = "Sanguine spell that enchants your next weapon strike to deal more damage, heal you for damage dealt, and refill blood."
+	spell_type =  /datum/action/cooldown/spell/sanguine_strike
+	category = "Offensive"
+
+/datum/spellbook_entry/scream_for_me
+	name = "Scream For Me"
+	desc = "Sadistic sanguine spell that inflicts numerous severe blood wounds all over the victim's body."
+	spell_type =  /datum/action/cooldown/spell/touch/scream_for_me
+	cost = 1
```

Comment:
should probably be made 2, since this is pretty much just a different version of smite that has your target instead die a little bit after. being able to empower this with only 5 points is probably going to lead  to shenanigans

Reply:
how delayed is the death, exactly? are we talking 5 seconds or 15?

Reply:
It does miss the aoe flash. I'll make it reduce for less per level to prevent chicanery.

Reply:
> how delayed is the death, exactly? are we talking 5 seconds or 15?

around 10?